longTrip.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+import math
+
+class Trip:
+    """
+    トリップ情報を保持するクラス
+    
+    Attributes
+    ----------
+    userId: int
+        ユーザID
+    points: dict
+        キーに座標のタイムスタンプ、値にXY座標
+    
+    """
+    
+    userId = -1
+    points = {}
+    
+    def __init__(self, userId):
+        """
+        初期化
+        
+        Attributes
+        ----------
+        userId: int
+            ユーザID
+        """
+        
+        self.userId = userId
+        self.points = {}
+    
+    def addPoint(self, timeStamp, pointX, pointY):
+        """
+        座標の追加
+        
+        Attributes
+        ----------
+        timeStamp: int
+            UNIXタイムでの時刻
+        pointX: int
+            X座標
+        pointY: int
+            Y座標
+        """
+        self.points[timeStamp] = (pointX, pointY)
+        
+    def print(self):
+        print("userId: " + str(self.userId))
+        print(self.points)
+        
+    def calcDistance(self):
+        """
+        当該トリップの移動距離算出
+        pointsデータを参照し、スタート地点からゴール地点までの距離を算出
+        """
+        distance = 0
+        lastX = None
+        lastY = None
+        for point in self.points:
+            curX = self.points[point][0]
+            curY = self.points[point][1]
+            if lastX != None:
+                curDistance = math.sqrt((curX - lastX)**2 + (curY - lastY)**2)
```

Comment:
座標間の距離計算は、Lineクラスを作成して、Lineクラスにcalc_lengthメソッドを実装すると低凝集にできると思います。


```suggestion
    class Line
        __init__(self, start: Coordinate, end: Coordinate):
            self.start = start
            self.end = end
        
        calc_length(self):
            return math.sqrt((self.start.x - self.end.x)**2 + (self.start.y - self.end.y)**2)
```